data/tabular/perovskite_db/meta.yaml

```diff
@@ -0,0 +1,155 @@
+---
+name: perovskite_db
+description: |-
+  Database of perovskite solar cells with their composition, device stacks, and performance.
+targets:
+    - id: bandgap
+      description: bandgap of the perovskite material
+      units: eV
+      type: float
```

Comment:
I guess you need to change `type: float` to `type: continous`?

Reply:
yep: https://github.com/OpenBioML/chemnlp/blob/main/src/chemnlp/data_val/model.py#L35

Reply:
addressed